base/util.go

```diff
@@ -64,12 +64,14 @@ func NewNonCancelCtx() NonCancellableContext {
 }
 
 // NewNonCancelCtx creates a new background context struct for operations that require a fresh context, with database logging context added
-func NewNonCancelCtxForDatabase(dbName string, dbConsoleLogConfig *DbConsoleLogConfig) NonCancellableContext {
-	dbLogContext := DatabaseLogCtx(context.Background(), dbName, dbConsoleLogConfig)
-	ctxStruct := NonCancellableContext{
+func NewNonCancelCtxForDatabase(parentCtx context.Context, dbConsoleLogConfig *DbConsoleLogConfig) NonCancellableContext {
+	ctx := getLogCtx(parentCtx)
+
+	dbLogContext := DatabaseLogCtx(context.Background(), ctx.Database, dbConsoleLogConfig)
```

Comment:
This change keeps the same function signature as 3.2 and doesn't lose db console log settings in the OIDC case above.

```suggestion
func NewNonCancelCtxForDatabase(parentCtx context.Context) NonCancellableContext {
	ctx := getLogCtx(parentCtx)

	dbLogContext := DatabaseLogCtx(context.Background(), ctx.Database, ctx.DbConsoleLogConfig)
```

Reply:
Agreed, I didn't see it because it had a different name than it does on main.